plaso/filters/parser_filter.py

```diff
@@ -0,0 +1,201 @@
+# -*- coding: utf-8 -*-
+"""Helper for parser and plugin filter expressions."""
+
+from __future__ import unicode_literals
+
+
+class ParserFilterExpressionHelper(object):
+  """Helper for parser and plugin filter expressions.
+
+  A parser filter expression is a comma separated value string that denotes
+  a list of parser names to include and/or exclude. Each entry can have
+  the value of:
+
+  * An exact match of a preset, which is a predefined list of parsers
+    (see data/presets.yaml for the list of predefined presets).
+  * A name of a single parser (case insensitive), e.g. msiecf.
+  * A glob name for a single parser, e.g. '*msie*' (case insensitive).
+  """
+
+  def __init__(self, presets_manager):
+    """Initializes a helper for parser and plugin filter expressions.
+
+    Args:
+      presets_manager (ParserPresetsManager): a parser preset manager.
+    """
+    super(ParserFilterExpressionHelper, self).__init__()
+    self._presets_manager = presets_manager
```

Comment:
I recommend removing this, and instead pass the manager to ExpandPresets (and the methods it calls). It removes some indirection, as well as the init method, and makes the whole class static. This also makes testing more straightforward.

Reply:
ack